lib/api/non_commercial_search_options.rb

```diff
@@ -1,40 +1,14 @@
 # frozen_string_literal: true
```

Comment:
This essentially returns this file to the state it was in prior to the start of faceted search work: https://github.com/GSA/search-gov/blob/560cea539002c568b59f1c0db6087916d3b46478/lib/api/non_commercial_search_options.rb